docs/README.md

```diff
@@ -5,7 +5,7 @@ or referenced here.  For example, please include links (with a small description
 any relevant websites, external references, and other cloud storages (e.g. Google Docs).
 
 * [Developer documentation](/docs/development.md)
-* [Deployment documentation](/docs/deployment.md)
+* [Deployment documentation](/docs/deploy)
```

Comment:
Dropped the "root" docs file in favour of a README

<img width="837" alt="Screen Shot 2022-06-16 at 10 22 37" src="https://user-images.githubusercontent.com/48086/174091395-b14eb48d-c6fb-4e59-8185-796d5e32a7e5.png">